### ISSUE TITLE
gazebo_attache_plugin - as dependency

### ISSUE THREAD

**Issue by ghost** [code-iai/longterm_fetch_and_place#29]
- needs to be compiled relative to the gazebo version which runs
- i propose to: 
  - have it as dependency and of "longterm_fetch_and_place"
  - from "longterm_fetch_and_place" load the plugin and update the "GAZEBO_PLUGIN_PATH"

**Comment by fairlight1337** [comment]
Yeah, maybe we can put it into [cram_gazebo](https://github.com/cram2/cram_gazebo) as that's also where @mpomarlan's visibility plugin resides; also, then it gets compiled rather than shipped as binary (which what is the case right now, which is super bad practice - courtesy to myself).

The repository in question is this: [gazebo_attache_plugin](https://github.com/fairlight1337/gazebo_attache_plugin/).

Thanks for putting together the issue.

**Comment by mpomarlan** [comment]
I'm assigning @bbrieber to this as well, because he had a helpful idea: https://github.com/code-iai/iai_gazebo_utils which is a repository for such gazebo plugins.

The plan is to move my gazebo_visibility_ros plugin there soon; I'd suggest we put gazebo_attache_plugin there as well.

**Comment by fairlight1337** [comment]
Good idea. I'll see after moving it there as well. Do we need to make ROS packages out of those to make them accessible via some Gazebo plugin path? Or can we have them available somehow differently?

I imagine that having them as ROS packages places the compiled shared objects somewhere in the `devel` folder; this is per definition searched by Gazebo, isn't it? That would safe us from having to define custom install targets.

**Comment by mpomarlan** [comment]
Having them as ROS packages seems the way to go. Both plugins use ROS anyway.

**Comment by fairlight1337** [comment]
True. I'll see after getting it in there if that's what we're doing.
